src/main/java/seedu/address/storage/JsonAdaptedNote.java

```diff
@@ -50,9 +62,15 @@ public boolean isClientNote() {
      *
      * @return The correct Note object being represented by this json note.
      */
```

Comment:
Should probably change this javadoc to match those of other `toModelType()` functions like JsonAdaptedClient etc.